core/templates/components/oppia-angular-root.component.ts

```diff
@@ -83,6 +83,7 @@ import { AnswerGroupObjectFactory } from
 import { AnswerStatsObjectFactory } from
   'domain/exploration/AnswerStatsObjectFactory';
 import { AppService } from 'services/app.service';
+import { AttributionService } from 'services/attribution.service';
```

Comment:
These. changes are not required anymore 🙂

Reply:
Thanks @srijanreddy98, can you please provide more context for why it isn't required.